### DISCUSSION TITLE
Where/When does trivy config pull its rules from when running trivy config on cdk cloud configurations?

### DISCUSSION THREAD

**Discussion by Marcusdclark12** [aquasecurity/trivy#4721] [Q&A] [answered]
### Question

Hello, currently I am implementing Trivy into our pipeline and i want to get a better understanding on how trivy works when it is scanning cloud formation files against the dfsec rule repo. Is this repo updated automatically  when trivy config runs? Like for example when i do trivy config does is do a git pull dfsec rules behind the scenes? Is there way to pull from a stable version on the dfsec rules and scan against that every time instead of the updated rules? I would appreciate any answers to these questions, Thank you.

### Target

AWS

### Scanner

Misconfiguration

### Output Format

JSON

### Mode

Standalone

### Operating System

linux

### Version

```bash
Version: 0.42.1
Policy Bundle:
  Digest: sha256:030345bff4845bae1c89450da85681bc25c0c40d2144a0d9db5254674d155298
  DownloadedAt: 2023-06-26 17:21:07.307811674 +0000 UTC
```

**Comment by simar7** [comment, marked as answer]
Trivy uses policy bundles, which are cached locally and fetched when need be. You can read more on this here https://aquasecurity.github.io/trivy/v0.42/docs/scanner/misconfiguration/policy/builtin/#policy-distribution

You can disable fetching the rules as described in the link above.
